src/vtbackend/ViInputHandler.h

```diff
@@ -100,6 +100,9 @@ enum class ViMotion : uint8_t
     ToCharLeft,            // F {char}
     RepeatCharMove,        // ;
     RepeatCharMoveReverse, // ,
+    JumpToLastJumpPoint,   // '' or `` (jump to last jump)
+    JumpToMarkUp,          // <C-O>
+    JumpToMarkDown,        // <C-I>
```

Comment:
I think that Up/Down is really misleading here. It should be either Out/In or Backward/Forward, which makes more sense in my opinion :)